packages/config-plugins/src/plugins/ios-plugins.ts

```diff
@@ -17,7 +21,7 @@ type MutateInfoPlistAction = (expo: ExpoConfig, infoPlist: InfoPlist) => InfoPli
 export function createInfoPlistPlugin(action: MutateInfoPlistAction, name?: string): ConfigPlugin {
   const withUnknown: ConfigPlugin = config =>
     withInfoPlist(config, async config => {
-      config.modResults = await action(config, config.modResults);
+      config.modResults = await action(config, config.modResults, config.modRequest);
```

Comment:
This changes the symmetry of the other methods, should just use `withInfoPlist` directly where needed.